src/compile/data.ts

```diff
@@ -215,6 +216,21 @@ export namespace source {
       return transform;
     }, []);
   }
+
+  export function rankTransform(model: Model) {
```

Comment:
Add comment that this add rank transform for the color ramp as a workaround (until the future color ramp support in Vega).